lib/MusicBrainz/Server/Data/Series.pm

```diff
@@ -171,7 +171,7 @@ sub update {
 }
 
 sub is_empty {
-    my ($self, $series_id) = @_;
+    my ($self, $series_id, $ignore_edits) = @_;
```

Comment:
Since I'm passing $ignore_edits I guess I should have it here even if it's unused. I'm a bit surprised that this doesn't actually look at pending / open edits at all.

Reply:
This comment seems to be outdated?